resources/guidelines/testing/store/quality-guidelines-plugins/index.md

```diff
@@ -487,23 +501,31 @@ The solution is to amend the requirement as follows:
 This ensures that at least version Shopware 6.1 is installed, even if it is a Release Candidate.
 It will be preferred as soon as the final 6.1 is released.
 
-### Unauthorized file formats or folders detected in the app. Please remove the following files/folders
-
-Not allowed folders and files:
-
-    * .gitignore
-    * .DS_Store
-    * Thumbs.db
-    * .git, __MACOSX
-    * .zip
-    * .tar
-    * .tar.gz
-    * .phar
-
-## Note on Shopware technology partner contract for interfaces
-
-You have now read the complete list of requirements for developing and releasing apps based on our app system in the Shopware Community Store.
-
-If your app is a software app/interface with downstream costs, transaction fees, or service fees for the customer, we need to complete a technology partner agreement in order to activate your apps.
-
-If you have any questions regarding the technology partner agreement, please contact our sales team by writing an email to [alliances@shopware.com](mailto:alliances@shopware.com) or calling **+44 (0) 203 095 2445 (UK) / 00 800 746 7626 0 (worldwide) / +49 (0) 25 55 / 928 85-0 (Germany)**.
+### Remove out-commented code from your source-code
+
+### Unauthorized file formats or folders detected in the app.
+
+Remove out-commented code, unused files and folders, and all dev-files from your binary.
+
+Here are some examples of not allowed folders and files:
+* /__MACOSX
+*./tests
+* .gitgignore
+* .gitkeep
+* .gitlab-ci.yml
+* .DS_Store
+* .editorconfig
+* .git
+* .gitignore
+* .shopware-extension.yml
```

Comment:
@shyim , when the `.shopware-extension.yml` becomes an unauthorized file. We can't use the shopware-cli to create zip files. Is this correct?

Reply:
that file neeeds to stay, otherwise we miss some information for `shopware-cli project ci`